src/teleop_twist_joy.cpp

```diff
@@ -80,7 +80,7 @@ TeleopTwistJoy::TeleopTwistJoy(rclcpp::node::Node::SharedPtr & node)
   rmw_qos_profile_t custom_qos_profile;
   custom_qos_profile.history = RMW_QOS_POLICY_HISTORY_KEEP_LAST;
   custom_qos_profile.depth = 50;
-  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT;
```

Comment:
Same here, we tried for all data stream the default sensor profile and had better results than with a queue size as big as 50. Can you perform the same test on your side before we make changes to the QoS on our "master"?